src/components/SpaceProposalPage.vue

```diff
@@ -64,8 +64,8 @@ function clickVote() {
       : (modalOpen.value = true);
 }
 
-function reloadProposal() {
-  emit('reload-proposal');
+function reloadProposal(softReload = false) {
```

Comment:
```suggestion
function reloadProposal({ softReload = false } = {}) {
```
Maybe doing like this?

`reloadProposal(true)` isn't very readable IMO

Reply:
If we also soft reload votes later, we could just move the loading status toggler to  onmounted, and loadproposal will always be a soft reload

Reply:
Ok we can do that. Should I merge this for now then?

Reply:
Yes